tests/integrations/backup/mod.rs

```diff
@@ -416,10 +422,132 @@ fn test_backup_rawkv() {
+    let storage = create_storage(&backend, Default::default()).unwrap();
+    let region = target_suit.cluster.get_region(b"");
+    let mut sst_meta = SstMeta::default();
+    sst_meta.region_id = region.get_id();
+    sst_meta.set_region_epoch(region.get_region_epoch().clone());
+    sst_meta.set_uuid(uuid::Uuid::new_v4().as_bytes().to_vec());
+    sst_meta.set_api_version(dst_api_ver);
+    let mut metas = vec![];
+    for f in files1.clone().into_iter() {
+        let mut reader = storage.read(&f.name);
+        let mut content = vec![];
+        block_on(reader.read_to_end(&mut content)).unwrap();
+        let mut m = sst_meta.clone();
+        m.crc32 = calc_crc32_bytes(&content);
+        m.length = content.len() as _;
+        m.cf_name = name_to_cf(&f.name).to_owned();
+        metas.push((m, content));
+    }
+
+    for (m, c) in &metas {
+        for importer in target_suit.cluster.sim.rl().importers.values() {
+            let mut f = importer.create(m).unwrap();
+            f.append(c).unwrap();
+            f.finish().unwrap();
+        }
+
+        // Make ingest command.
+        let mut ingest = Request::default();
+        ingest.set_cmd_type(CmdType::IngestSst);
+        ingest.mut_ingest_sst().set_sst(m.clone());
+        let mut header = RaftRequestHeader::default();
+        let leader = target_suit.context.get_peer().clone();
+        header.set_peer(leader);
+        header.set_region_id(target_suit.context.get_region_id());
+        header.set_region_epoch(target_suit.context.get_region_epoch().clone());
+        let mut cmd = RaftCmdRequest::default();
+        cmd.set_header(header);
+        cmd.mut_requests().push(ingest);
+        let resp = target_suit
+            .cluster
+            .call_command_on_leader(cmd, Duration::from_secs(5))
+            .unwrap();
+        assert!(!resp.get_header().has_error(), "{:?}", resp);
+    }
+
+    // Backup file should have same contents.
+    // Set non-empty range to check if it's incorrectly encoded.
+    let rx = target_suit.backup_raw(
+        vec![b'r', b'a'], // start
+        vec![b'r', b'z'], // end
+        cf,
+        &make_unique_dir(tmp.path()),
+        dst_api_ver,
+    );
+    let resps3 = block_on(rx.collect::<Vec<_>>());
+    let files3 = resps3[0].files.clone();
+
+    // After https://github.com/tikv/tikv/pull/8707 merged.
+    // the backup file name will based on local timestamp.
+    // so the two backup's file name may not be same, we should skip this check.
+    assert_eq!(files1.len(), 1);
+    assert_eq!(files3.len(), 1);
+    assert_eq!(files1[0].total_bytes, files3[0].total_bytes);
+    assert_eq!(files1[0].total_kvs, files3[0].total_kvs);
+    suite.stop();
+    target_suit.stop();
+}
+
+#[test]
+fn test_backup_rawkv_v1_to_v2() {
+    test_backup_rawkv_convert_impl(ApiVersion::V1, ApiVersion::V2)
+}
+
+fn test_backup_raw_meta_impl(cur_api_version: ApiVersion, dest_api_version: ApiVersion) {
```

Comment:
Suggest use same naming for `cur_api_ver(ion)` & `d(e)st_api_ver(ion)`.

Reply:
done